src/OrchardCore/OrchardCore.Localization.Abstractions/NullHtmlLocalizerOfT.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using Microsoft.AspNetCore.Mvc.Localization;
+using Microsoft.Extensions.Localization;
+
+namespace OrchardCore.Localization
+{
+    /// <summary>
+    /// Minimalistic HTML-aware localizer that does nothing.
+    /// </summary>
+    /// <typeparam name="T">The <see cref="Type"/> to provide strings for.</typeparam>
+    public class NullHtmlLocalizer<T> : IHtmlLocalizer<T>
```

Comment:
Here, I'm not sure if it's necessary to have a NullHtmlLocalizer of T since in the end it is just something to tell that there is no localizer available. Why would we need to pass a Type? The only usage to me would be if we had non-localizable types?

Reply:
Seems there's no need for `NullHtmlLocalizer<T>` at all